src/test/java/uk/gov/cabinetoffice/csl/service/ModuleRollupServiceTest.java

```diff
@@ -2,134 +2,97 @@
-        catalogueModule.setId("moduleId2");
-        catalogueCourse.getModules().add(catalogueModule);
-        mockLearningCatalogueServiceForGetCachedCourse(catalogueCourse);
-        CourseRecord updatedCourseRecord = invokeService();
-        verify(updatedCourseRecord, State.IN_PROGRESS);
-    }
-
-    private CourseRecord invokeService() {
-        return moduleRollupService.processRusticiRollupData(rusticiRollupData);
-    }
-
-    private void verify(CourseRecord updatedCourseRecord, State expectedState) {
-        assertNotNull(updatedCourseRecord);
-        assertEquals(courseId, updatedCourseRecord.getCourseId());
-        assertEquals(rusticiRollupData.getUpdated(), updatedCourseRecord.getLastUpdated());
-        assertEquals(expectedState, updatedCourseRecord.getState());
-
-        ModuleRecord updatedModuleRecord = updatedCourseRecord.getModuleRecord(moduleId);
-        assertNotNull(updatedModuleRecord);
-        assertEquals(moduleId, updatedModuleRecord.getModuleId());
-        assertEquals(rusticiRollupData.getUpdated(), updatedModuleRecord.getUpdatedAt());
-        assertEquals(rusticiRollupData.getUpdated(), updatedModuleRecord.getCompletionDate());
-        assertEquals(State.COMPLETED, updatedModuleRecord.getState());
-        assertEquals(Result.PASSED, updatedModuleRecord.getResult());
+    public void emptyPatchDataShouldNotBeProcessed() {
+        mockGetCSLDataFromRollup(new CSLRusticiProps("", "", "", List.of()));
+        invokeService();
+        verify(learnerRecordService, never()).getCourseRecord("", "");
+        verify(learnerRecordService, never()).updateModuleRecord(any(), any());
+        verify(learnerRecordService, never()).updateCourseRecord(any(), any(), any());
     }
 
-    private void mockCourseAndModuleState(CourseRecord courseRecord, State courseState) {
-        Map<String, String> updateFields = new HashMap<>();
-        updateFields.put("updatedAt", rusticiRollupData.getUpdated().toString());
-        updateFields.put("completionDate", rusticiRollupData.getCompletedDate().toString());
-        updateFields.put("state", State.COMPLETED.name());
-        updateFields.put("result", rusticiRollupData.getRegistrationSuccess());
-
-        ModuleRecord moduleRecord = courseRecord.getModuleRecord(moduleId);
-        moduleRecord.setUpdatedAt(rusticiRollupData.getUpdated());
-        moduleRecord.setCompletionDate(rusticiRollupData.getCompletedDate());
-        moduleRecord.setState(State.COMPLETED);
-        moduleRecord.setResult(Result.PASSED);
-        cslTestUtil.mockLearnerRecordServiceForUpdateModuleRecord(moduleRecord.getId(), updateFields, moduleRecord);
-
-        courseRecord.setState(courseState);
-        courseRecord.setLastUpdated(rusticiRollupData.getUpdated());
-        cslTestUtil.mockLearnerRecordServiceForUpdateCourseRecordState(
-                learnerId, courseId, State.COMPLETED, rusticiRollupData.getUpdated(), courseRecord);
+//    @Test
```

Comment:
Could we please remove the commented code if not required for any future purpose?